File: Practice/05/C++/5.cpp

#include <iostream>

int main()
{
    setlocale(LC_ALL, "Russian");
    int x0, v0, t, xt;
    std::cin >> x0;
    std::cin >> v0;
    std::cin >> t;
    xt = x0 + (v0 * t) + ((9.8 * t * t) / 2);
    std::cout << abs(xt - x0);
}

File: Practice/05/Python/5.py
x0 = float(input('x(0) = ')) # Лучше выбрать тип float,ибо данные могут быть дробными
v0 = float(input('v(0) = '))
t = float(input('t = '))
xt = (x0 + (v0 * t)) - ((9.8 * t * t) / 2) # Разници особой между вариантами нет
print(abs(xt - x0))
